sormas-api/src/main/resources/strings.properties

```diff
@@ -1125,42 +1141,77 @@ messageEventParticipantDearchived = Event participant has been de-archived
+messageCountEntitiesNotEditedAccessDeniedReason = %s entities not edited because the entity is not editable anymore: %s
+messageCountEntitiesNotSent = %s entities can not be sent to the reporting tool: %s
+messageCountEntitiesNotSentAccessDeniedReason = %s entities can not be sent to the reporting tool because the user does not have the proper rights: %s
+messageCountEntitiesNotSentExternalReason = %s entities can not be sent to the reporting tool because the communication with the reporting tool failed: %s
+messageCountEventsNotArchivedExternalReason = %s events not archived because the communication with the reporting tool failed: %s
+messageCountEventsNotDearchivedExternalReason = %s events not dearchived because the communication with the reporting tool failed: %s
+messageCountEventsNotLinked = %s events can not be linked or they are already linked to the event group: %s
+messageCountEventsNotLinkedAccessDeniedReason = %s events can not be linked to the event group because is not allowed to link events from another region to an event group.: %s
 messageCountEventsNotRestored = %s events not restored. UUIDs of events not restored: %s
-messageCountEventParticipantsNotDeleted = %s event participants not deleted. UUIDs of event participants not deleted: %s
+messageCountEventParticipantsNotDeleted = %s event participants not deleted: %s
+messageCountEventParticipantsNotDeletedAccessDeniedReason = %s event participants not deleted because they are not in jurisdiction or owned: %s
 messageCountEventParticipantsNotRestored = %s event participants not restored. UUIDs of event participants not restored: %s
-messageCountExternalMessagesNotDeleted = %s external messages not deleted. UUIDs of external messages not deleted: %s
+messageCountExternalMessagesNotDeleted = %s external messages not deleted: %s
+messageCountImmunizationsNotDeleted = %s immunizations not deleted: %s
+messageCountImmunizationsNotDeletedAccessDeniedReason =%s immunizations not deleted because they are not in jurisdiction or owned: %s
 messageCountImmunizationsNotRestored = %s immunizations not restored. UUIDs of immunizations not restored: %s
-messageCountSamplesNotDeleted = %s samples not deleted. UUIDs of samples not deleted: %s
+messageCountSamplesNotDeleted = %s samples not deleted: %s
 messageCountSamplesNotRestored = %s samples not restored. UUIDs of samples not restored: %s
-messageCountTasksNotDeleted = %s tasks not deleted. UUIDs of travel entries not deleted: %s
-messageCountTravelEntriesNotDeleted = %s travel entries not deleted. UUIDs of travel entries not deleted: %s
+messageCountTasksNotDeleted = %s tasks not deleted: %s
+messageCountTasksNotDeletedAccessDeniedReason =%s tasks not deleted because the they are not in jurisdiction or owned: %s
+messageCountTravelEntriesNotDeleted = %s travel entries not deleted: %s
+messageCountTravelEntriesNotDeletedAccessDeniedReason =%s travel entries not deleted because the they are not in jurisdiction or owned: %s
```

Comment:
```suggestion
messageCountTravelEntriesNotDeletedAccessDeniedReason = %s travel entries not deleted because the they are not in jurisdiction or owned: %s
```